quotacontrol.go

```diff
@@ -158,16 +158,44 @@ func (q qcHandler) GetAccessKey(ctx context.Context, accessKey string) (*proto.A
 	return q.accessKeyStore.FindAccessKey(ctx, accessKey)
 }
 
+func (q qcHandler) GetDefaultAccessKey(ctx context.Context, projectID uint64) (*proto.AccessKey, error) {
+	list, err := q.accessKeyStore.ListAccessKeys(ctx, projectID, proto.Ptr(true), nil)
+	if err != nil {
+		return nil, err
+	}
+
+	var defaultAccessKey *proto.AccessKey 
+	for _, accessKey := range list {
+		if accessKey.Default {
+			defaultAccessKey = accessKey
+			break
+		}
+	}
+
+	if defaultAccessKey == nil {
+		return nil, fmt.Errorf("no default access key found")
+	}
+
+	return defaultAccessKey, nil
```

Comment:
Add Error in ridl and do:

```suggestion
	for _, accessKey := range list {
		if accessKey.Default {
			return accessKey, nil
		}
	}

	return nil, proto.ErrNoDefaultKey
```